_posts/2024-04-15-gsi-tech-blog.md

```diff
@@ -0,0 +1,75 @@
+
+The number of satellites is increasing. We have five times more satellites in space than we did ten years ago. The proportional amount of humans we have to do this processing is decreasing. Humans have become a bottleneck.
+
+![](/assets/media/blog-images/2024-04-15-gsi-tech-blog/objects-launched-annual.jpg)
+
+## What if...
+
+Suppose these types of satellites could be outfitted with a card that both stored and computed, equivalent to having both RAM and CPU, with the capability to run their sensor data through an ML model. The goal being to vectorize and pre-process sensor data. Sending a vector or change detection earthside is a much quicker operation than sending a complete set of sensor data. One of our partners, [GSI Technology](https://gsitechnology.com), might just change some of the landscape in this regard. They’ve introduced a new kind of chip into the market. The project, called “Gemini-II”, is an APU. It is an extremely power efficient processing unit that can both store and compute in its own distributed memory. You may have seen their booth at [OpenSearchCon 2023](http://www.opensearch.org/events/opensearchcon/2023/north-america/index.html).
+
+The APU technology family is a Compute-in-memory (CiM) architecture seeking to eliminate computation time wasted on data movement by closing the traditional gap between processors (millions in GSI's case) and storage elements, which is highly beneficial for data-intensive workloads. Computation occurs where the data lives - in the storage elements, avoiding traditional Processor/Memory bottlenecks!
+
+## Several applications
+
+Many institutions often use things like SAR (synthetic aperture radar) to monitor landscapes to check for things like the progress of deforestation, forest fires, as well as the erosion of coastal areas. Even corporations use this tech to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use this tech to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.
+
+If you’re curious about what GSI Technology works on you should definitely head over to their [website](https://gsitechnology.com). They’re applying AI and ML to a variety of modern problems. There are many potential applications that are up to their imagination - aerospace, genomics, computer vision, cyber security for space, vector search in space, data centers in space for the purposes of computing large volumes of data and more.
+
+## Parting food for thought
```

Comment:
Try not to use idiomatic expressions, as they will likely not be understood by portions of our audience.